src/aiida/__init__.py

```diff
@@ -35,6 +42,8 @@
 )
 __paper_short__ = 'S. P. Huber et al., Scientific Data 7, 300 (2020).'
 
+__all__ = ['load_profile', 'configure_logging', 'get_config_option', 'get_profile', 'profile_context']
```

Comment:
I think it makes no sense to expose `configure_logging` to end user. But for backward compatibility, I keep it for the moment. I prefer to remove it, agree?

For `load_profile` and `profile_context` and `get_config_option`, they are mentioned in the documentation so I keep those. The `get_profile` also seems quite handy.

Reply:
In order for this to not be a breaking change, you'll have to include `get_strict_version` and `load_ipython_extension` as well.

Reply:
Yes, but if they did `from aiida import *` than now suddenly `get_strict_version` will not be available.

Reply:
In other words, I believe when `__all__` is not defined, all symbols are imported when you do start import.

Reply:
I tested, that's true, I didn't know before. Then I'll add them to `__all__`.